README.md

```diff
@@ -16,17 +34,55 @@ Generated documentation for this SDK can be found at https://rakutentech.github.
 
 To run the example project, clone the repo, and run `pod install` from the Example directory first.
 
-## Requirements
+### Custom parameters
+
+In order to load your MiniApp, you will have to use your own Host App ID and your subscription key. These can eiher be set in project configuration plist (RASApplicationIdentifier and RASProjectSubscriptionKey) or by taping the top right configuration icon in the example application.
 
 ## Installation
 
-MiniApp is available through [CocoaPods](https://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+MiniApp is available through [CocoaPods](https://cocoapods.org). To install it, simply add the following line to your Podfile:
 
 ```ruby
 pod 'MiniApp'
 ```
 
+In your project configuration .plist you can add 2 custom string target properties :
+
+- `RASApplicationIdentifier` - to set your MiniApp host application identifier
+- `RASProjectSubscriptionKey` - to set your MiniApp subsbscription key
+
+If you don't want to use project settings, you are still able to pass this informations one by one to the `Config.userDefaults` using a `Config.Key` as key:
+
+```
+Config.userDefaults?.set("MY_CUSTOM_ID", forKey: Config.Key.subscriptionKey.rawValue)
+```
+
+## Usage
+
+MiniApp library calls are done via the `MiniApp.shared()` singleton with or without a `MiniAppSdkConfig` instance (you can get the current one with `Config.getCurrent()`). If you don't provide a config instance, values in custiom iOS target properties will be used by default. For example, if you want to load the MiniApp list:
+
+```
+MiniApp.shared(with: Config.getCurrent()).list { (result) in
+	...
+}
+```
+
+Getting a `MiniAppInfo` :
+
+```
+MiniApp.shared().info(miniAppId: miniAppID) { (result) in
+	...
+}
+```
+
+Create a MiniApp for the given `MiniAppInfo` object :
+
+```
+MiniApp.shared().create(appInfo: info, completionHandler: { (result) in
+	...
```

Comment:
I think the example for `create` should be a little more complete.  On Android side, there is mostly complete Activity example showing how they would set the mini app to the view after retrieving it.